src/app/components/mayastor/mayastor.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardData } from 'src/app/services/ci-dashboard.service';
+
+@Component({
+  selector: 'app-mayastor',
+  templateUrl: './mayastor.component.html',
+  styleUrls: ['./mayastor.component.css']
+})
+export class MayastorComponent implements OnInit {
+  data: any;
+  constructor(private ApiService: DashboardData,) {
+    ApiService.getMayastorTest().subscribe(res => {
+      this.data = res;
+    }, err => { console.log('Unable to get mayastor endpoint', err); });
```

Comment:
Let's please write well formatted and readable code. For example, if we wrap parenthesis for the err here, that would be more readable. Feature wise, it works fine, but let's think about the next developer who will be working on this code base.